packages/react-core/src/components/Form/FormGroup.tsx

```diff
@@ -20,6 +20,8 @@ export interface FormGroupProps extends Omit<React.HTMLProps<HTMLDivElement>, 'l
   validated?: 'success' | 'error' | 'default';
   /** Sets the FormGroup isInline. */
   isInline?: boolean;
+  /** Removes top spacer from label. */
+  isNoPaddingTop?: boolean;
```

Comment:
I would change either change this name to `hasNoPaddingTop` or I would call is `hasPaddingTop` and and set the default to `True`.  The second option would be my preference.

@jschuler do you have preference here?

Reply:
I left a comment about this in a drawer PR - https://github.com/patternfly/patternfly-react/pull/4036, and mentioned how the `<PageSection>` supports `noPadding`, and `noPaddingMobile`.